src/core/lib/gpr/arena.cc

```diff
@@ -33,6 +33,8 @@
 #include "src/core/lib/gprpp/memory.h"
 
 namespace {
+enum class ArenaAlignment { MaxRequired, CacheLine };
```

Comment:
We're still using C-style CAPITAL_SNAKE_CASE enums everywhere, afaik

Reply:
Ack, fixed.